coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -222,6 +225,31 @@ private void parseExcludeNodesFile(DataInputStream fsDataInputStream) throws IOE
         "Updated exclude nodes and {} nodes were marked as exclude nodes", excludeNodes.size());
   }
 
+  private void putInExcludeNodesFile(List<String> excludeNodes) throws IOException {
+    StringBuilder appendExecludeNodes = new StringBuilder();
+    String currentDate = DateFormatUtils.format(new Date(), "yyyy/MM/dd HH:mm:ss");
```

Comment:
```suggestion
    String currentDate = DateFormatUtils.format(System.currentTimeMillis(), DATE_PATTERN);
```

And `
  private static final String DATE_PATTERN = "yyyy-MM-dd HH:mm:ss";`

Reply:
Ready.